ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.cy.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { ApolloProvider } from '@apollo/client';
+import { createBrowserHistory } from 'history';
+
+import configureApolloClient from 'configureApolloClient';
+import { standardEntityTypes } from 'constants/entityTypes';
+import { complianceBasePath, urlEntityListTypes } from 'routePaths';
+import ComplianceLevelsByStandard from './ComplianceLevelsByStandard';
+
+/*
+These standards have been formatted for easier verification of the expected ordering in
+tests compared to a direct hard coding in the mocked response below.
+
+id: [name, numFailing, numPassing]
+*/
+const standards = {
+    CIS_Kubernetes_v1_5: ['CIS Kubernetes v1.5', 8, 2],
+    HIPAA_164: ['HIPAA 164', 7, 3],
+    NIST_800_190: ['NIST SP 800-190', 6, 4],
+    NIST_SP_800_53_Rev_4: ['NIST SP 800-53', 5, 5],
+    PCI_DSS_3_2: ['PCI DSS 3.2.1', 4, 6],
+    'ocp4-cis': ['ocp4-cis', 3, 7],
+    'ocp4-cis-node': ['ocp4-cis-node', 2, 8],
+};
+
+const mock = {
+    data: {
+        controls: {
+            results: Object.entries(standards).map(([id, [, numFailing, numPassing]]) => ({
+                aggregationKeys: [{ id, scope: 'STANDARD' }],
+                numFailing,
+                numPassing,
+                numSkipped: 0,
+                unit: 'CONTROL',
+            })),
+        },
+        complianceStandards: Object.entries(standards).map(([id, [name]]) => ({
+            id,
+            name,
+        })),
+    },
+};
+
+beforeEach(() => {
+    localStorage.clear();
```

Comment:
This is copy from Jest test, correct?

Worth investigation to see whether still needed for cypress component tests.

https://docs.cypress.io/api/commands/clearlocalstorage

> Cypress automatically clears all local storage before each test to prevent state from being shared across tests when [test isolation](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Test-Isolation) is enabled. You shouldn't need to use this command unless you're using it to clear localStorage inside a single test or test isolation is disabled.